web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
> but extending this practice across all your contributions will lead to bloated files like the Odoo ones

@pedrobaeza, you're right, I don't want to have bloated files as Odoo. Then if your are OK, I can make a PR to define a way to express guidelines more comfortable for everyone on this matter.